src/fidesctl/api/ctl/database/database.py

```diff
@@ -87,10 +87,23 @@ async def load_default_taxonomy() -> None:
     upsert_resource_types = list(DEFAULT_TAXONOMY.__fields_set__)
     upsert_resource_types.remove("organization")
 
-    log.info("UPSERTING the remaining default fideslang taxonomy resources")
+    log.info("INSERTING new default fideslang taxonomy resources")
     for resource_type in upsert_resource_types:
         log.info(f"Processing {resource_type} resources...")
-        resources = list(map(dict, DEFAULT_TAXONOMY.dict()[resource_type]))
+        default_resources = DEFAULT_TAXONOMY.dict()[resource_type]
+        default_keys = {item["fides_key"] for item in default_resources}
+        existing_resources = await list_resource(sql_model_map[resource_type])
+        existing_keys = {item.fides_key for item in existing_resources}
+        new_default_keys = default_keys.difference(existing_keys)
+        resources = [
+            resource
+            for resource in default_resources
+            if resource["fides_key"] in new_default_keys
+        ]
+
+        if len(resources) == 0:
+            log.info(f"No new {resource_type} resources to add from default taxonomy.")
+            continue
 
         try:
             result = await upsert_resources(sql_model_map[resource_type], resources)
```

Comment:
as an additional safeguard, should we change this as well to `create_resource`?

Reply:
so it'll fail if there is somehow a bug and we're about to overwrite a user-defined taxonomy?

Reply:
yeah that makes sense! the only thing is `upsert_resources` lets you bulk add while `create_resource` is just one at a time. it'd probably be easy to make a `create_resources`? or could for loop the `create_resource` since we probably aren't adding all that many things to the default taxonomy at once, and this function only runs on startup? thoughts?

Reply:
I think throwing them in a loop is fine! Startup latency isn't a huge issue at the moment